diagnostics-extension/src/controllers/events.controller.ts

```diff
@@ -75,11 +74,7 @@ export const registerEventHandlers = async () => {
 };
 
 export const onEventPortConnect = async (port: chrome.runtime.Port) => {
-  if (port.name === PortName.EVENTS_PORT) {
-    eventsService.registerPort(port);
-  } else {
-    console.error(ResponseErrorInfoMessage.InvalidPortName);
-  }
+  eventsService.registerPort(port);
```

Comment:
How about `eventsService.registerPort(PortName.EVENTS_PORT)`?

Reply:
Ditto to routine-v2 controller.

Reply:
Hmmm I don't think this comment applies to the change here.

For this particular code snippet, I've moved the port-name checking statement outside of the scope of this function.

Instead, the port name check is now performed at root-level in `sw.ts` file.

registerPort here is used to pass and store the received port object into the relevant service (eventService or routineV2Service). So we must still pass the port object.

Reply:
Ahh I misunderstood the usage of `port` argument. This patch looks good to me.